python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cleaning_strategy.py

```diff
@@ -106,15 +106,15 @@
 #
 #   \mathcal{A}^{d,j} = \left\{ \boldsymbol{\alpha} \in \mathbb{N}^p
 #   \; | \; \|\boldsymbol{\alpha}\|_1 \leq d, \;
-#   \; \|\boldsymbol{\alpha}\|_0 \leq j\right\}.
+#   \; \|\boldsymbol{\alpha}\|_0 \leq j\right\}
```

Comment:
```suggestion
#   \; \|\boldsymbol{\alpha}\|_0 \leq j\right\}.
```

This is a wrong suggestion because the point ends the sentence. Please consider for example eq. 1 page 6 in [^Knuth]:

![image](https://github.com/openturns/openturns/assets/31351465/a117d891-9f7c-4f17-9b72-d7d414002f9e)


[^Knuth]: Donald E. Knuth, Tracy Larrabee, and Paul M. Roberts. Mathematical Writing. Available at [this link](https://jmlr.csail.mit.edu/reviewing-papers/knuth_mathematical_writing.pdf).